### DISCUSSION TITLE
Recommendation

### DISCUSSION THREAD

**Discussion by eiung** [tailwindlabs/tailwindcss#14474] [Ideas]
When writing many attributes for pseudo-elements, what is a convenient way to add them? For example: after:w-4 after:h-4 after:absolute after:left-3 is written as after:(w-4, h-4, absolute, left-3)

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Comment by mary598chess** [comment]
Hello,
When dealing with multiple attributes [mcdvoice](https://www.mcdvoice.site) for pseudo-elements, a convenient method is to group them together. You can write after:(w-4, h-4, absolute, left-3) to keep your code cleaner and more organized.

Best Regards,
Mary Chess
